operator/cmd/cilium_node.go

```diff
@@ -42,7 +42,7 @@ func (c *ciliumNodeName) GetKeyName() string {
 	return nodeTypes.GetKeyNodeName(c.cluster, c.name)
 }
 
-type ciliumNodeManagerQueueSyncedKey struct{}
+const ciliumNodeManagerQueueSyncedKey = "_ciliumNodeManagerQueueSynced"
```

Comment:
I think its safe to assume that this will never be an actual key, as Kubernetes object names can never start with an underscore

Reply:
According to https://kubernetes.io/docs/concepts/overview/working-with-objects/names/ that seems to be the case. Though, perhaps it would also be good to explain this with a comment since its currently just a magic value.

Reply:
Added a comment to explain